api-catalog-package/src/main/resources/bin/validate.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019, 2020
+################################################################################
+
+INITIAL_ERRORS_FOUND=$ERRORS_FOUND
+
+# - GATEWAY_PORT - should not be bound to a port currently
+. ${ROOT_DIR}/bin/utils/network-utils.sh
+validate_port_is_available ${CATALOG_PORT}
+validate_host_is_resolvable "ZOWE_EXPLORER_HOST" # Note - takes variable name, not value as parameter
```

Comment:
I have mixed feelings of the network validations. Few thoughts:
- the validation method (onetstat, oping) is not compatible with Linux,
- in some extreme strict docker images, people may not have netstat/ping network util tools installed,
- the variable is validated here `ZOWE_EXPLORER_HOST` could be complicated. Currently it's simple and pointed to external domain where we access the system. But actually the system itself may not be necessary to have to resolve the domain correctly. Resolving the domain is only critical to external, the client side, which we cannot validate here,
- I have concerns to reference `ZOWE_EXPLORER_HOST` from component level. I think we should try to eliminate it as much as possible because the variable itself is confusing and has been used for various purpose: external domain, internal domains, referrer, etc.

Reply:
I don't disagree on principle, but this is al stuff that we've been using for years and there aren't alternatives yet. I don't understand what you want me to do, just park this PR until version 2!?

Reply:
I would suggest to remove the resolvable check. If we really want it, we can validate in run-zowe.sh so it's easier for us to migrate. Even in v2, I wouldn't expect every component to validate `haInstance.domain` value. Not sure how you think.

Reply:
I've removed the valdiation, but I'm not sure how that helps given that the start scripts all still use that variable.